client.py

```diff
@@ -95,28 +110,17 @@ def basic_query(self, expanded: bool = False) -> Dict:
         }
         return default
 
-    def overview_query(self):
+    def aggregator_query(self, *aggs: "QueryBuilder.Aggregators") -> Dict:
         query = self.basic_query()
         query.update(
             {
-                "aggregations": {
-                    "daily": {
-                        "date_histogram": {
-                            "field": "publication_date",
-                            "calendar_interval": "day",
-                            "min_doc_count": 1,
-                        }
-                    },
-                    "lang": {"terms": {"field": "language.keyword", "size": 100}},
-                    "domain": {"terms": {"field": "canonical_domain", "size": 100}},
-                    "tld": {"terms": {"field": "tld", "size": 100}},
```

Comment:
Are we dropping `tld`?

Reply:
None of the other layers above this point in the API use TLD- we had been calculating it here, I think, so that we had a backup for when the total count was above the 10,000 document limit imposed by elasticsearch- and we can replace that function with the domain field- so I removed it here in the spirit of speeding up the query! But, happy to consider leaving it in- @rahulbot any thoughts?

Reply:
I'm going to put this snippet of a conversation I had with @ibnesayeed many months ago here for context- 

> 
Basically, we had an aggregation on TLDs, so it would report how many matching documents are there for each TLD, which we can sum up to get the better total count. While we had aggregation on other aspects, such as language, date, etc., which I could have used for summation, but I chose TLD for two reasons: 1) it is finite in number, so there are very few entries to add and will never overflow the usual search result limit, and 2) almost all documents will have TLDs associated, unlike language or publication date, which might be empty, hence some documents would go uncounted.

I think this property of TLDs, where it is always filled, is also true of canonical_domain- so we can drop that in instead.

Reply:
Implemented this backup in the most recent commit